ops/docker/Dockerfile.integration-tests

```diff
@@ -4,7 +4,7 @@ FROM ${LOCAL_REGISTRY}/ethereumoptimism/builder:${BUILDER_TAG} AS builder
 
 FROM node:14-alpine
 
-RUN apk add --no-cache git curl python bash jq
+RUN apk add --no-cache git curl python3 bash jq
```

Comment:
I think it would be best to have this diff in a different PR if it's necessary for some reason

Reply:
Agree, that's my plan, I just pushed this to see if it would fix it on GHA too.